File: Sources/05XXX/5543/5543.pas
(*
 *  BOJ 5543번 Pascal 소스 코드
 *  작성자 : 동동매니저 (DDManager)
 *
 *  ※ 실행 결과
 *  사용 메모리 : 340 KB / 131,072 KB
 *  소요 시간 : 0 ms / 1,000 ms
 *
 *  Copyright 2019. DDManager all rights reserved.
 *)

VAR B,D,I,T:LONGINT;
BEGIN
	B:=2000;D:=2000;
	FOR I:=1 TO 5 DO
	BEGIN
		READ(T);
		IF(I<=3)AND(T<B)THEN B:=T
		ELSE IF(T<D)THEN D:=T;
	END;
	WRITE(B+D-50);
END.

File: Sources/05XXX/5543/5543.c
/**
 *  BOJ 5543번 C언어 소스 코드
 *  작성자 : 동동매니저 (DDManager)
 *
 *  ※ 실행 결과
 *  사용 메모리 : 1,116 KB / 131,072 KB
 *  소요 시간 : 0 ms / 1,000 ms
 *
 *  Copyright 2019. DDManager all rights reserved.
 */

#include <stdio.h>

int main(void){
	int b=2000,d=2000,i,t;
	for(i=0;i<5;i++){
		scanf("%d",&t);
		if(i<3&&t<b) b=t;
		else if(t<d) d=t;
	}
	printf("%d\n",b+d-50);
	return 0;
}